packages/ndla-video-search/src/VideoListItem.tsx

```diff
@@ -0,0 +1,166 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { useState } from "react";
+import { CloseLine } from "@ndla/icons/action";
+import { PanoramaPhotosphere } from "@ndla/icons/common";
+import { getLicenseByNBTitle, LicenseLocaleType } from "@ndla/licenses";
+import { Image, Text, ListItemContent, ListItemRoot, Button, IconButton } from "@ndla/primitives";
+import { styled } from "@ndla/styled-system/jsx";
+import { BrightcoveApiType } from "@ndla/types-embed";
+import { VideoTranslations } from "./VideoSearch";
+
+const ButtonWrapper = styled("div", {
+  base: {
+    display: "flex",
+    gap: "small",
+  },
+});
+
+const PreviewIframe = styled("iframe", {
+  base: {
+    minHeight: "surface.xxsmall",
+    gridArea: "center",
+  },
+});
+
+const PreviewWrapper = styled("div", {
+  base: {
+    display: "grid",
+    gridTemplateColumns: "repeat(3,1fr)",
+    gridTemplateAreas: `". center right"`,
```

Comment:
Går det ikke an å få til dette uten template-areas?

Reply:
Joda, det gjør det nok, kan eventuelt løse det med flex også la den venstre kolonnen være en tom div bare! Det er mulig man kan gjøre det på en annen måte med grid og, men det må jeg undersøke mer isåfall